compose/src/main/java/com/bottlerocketstudios/compose/repository/FileBrowserMockData.kt

```diff
@@ -0,0 +1,59 @@
+package com.bottlerocketstudios.compose.repository
+
+import androidx.compose.runtime.mutableStateOf
+
+internal val textString: String = "# These are some examples of commonly ignored file patterns.\n" +
```

Comment:
@VanyaHuamanBottleRocket Make this `private` if only used in this file OR refactor the name to be more specific and mentioning that it is mock data if it is used in other files